### PR TITLE
Akka 2.3.15

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: news/_posts/2016-04-01-akka-2.3.15-released.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+*Dear hAkkers,*
+
+We—the Akka committers—are pleased to be able to announce the availability of Akka 2.3.15.
+This is the 15th maintenance release of the 2.3 branch.
+This release features fixes that have been gathered since the previous release.
+
+Akka 2.3.15 is released for Scala 2.10 and 2.11. This release is backwards binary compatible with all previous 2.3.x versions which means that the new JARs are a drop-in replacement for the old one (but not the other way around) as long as your build does not enable the inliner (Scala-only restriction).
+Always make sure to use at least the latest version required by any of your project’s dependencies.
+
```
Comment (drewhk) [inline news/_posts/2016-04-01-akka-2.3.15-released.md]: maybe some highlights:

 - Various fixed leaks 
   - Nepotism in AbstractNodeQueue causing excessive garbage in old-gen
   - Non-closed Gzip streams in Clustering, causing off-heap memory leak
   - memory leak in AddressTerminated topic
   - remoting leaks actors if many systems join and leave
 - Fix issue causing mismatched acknowledge numbers causing restarted systems to be Quarantined
 - More robust handling of hanging connections in remoting

**Comment by drewhk** [comment]
LGTM but needs a bit more detail. I use these summaries in the news when supporting old versions to see which major issues where fixed since.

File: news/_posts/2016-04-01-akka-2.3.15-released.md (hunk 1 above)
Comment (2m) [inline news/_posts/2016-04-01-akka-2.3.15-released.md]: Thanks.

Comment (patriknw) [inline news/_posts/2016-04-01-akka-2.3.15-released.md]: I agree, such list would be good

**Comment by 2m** [comment]
Added highlights from Endre and also added links to issues/prs next to every highlight.

**Comment by patriknw** [comment]
LGTM

**Pull request merged**